app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -800,88 +757,72 @@ class DiscoveryView extends React.Component {
   };
 
   refreshMapPreviewedSamples = async () => {
-    const { domain } = this.props;
-    const {
-      mapLocationData,
-      mapPreviewedLocationId,
-      projectId,
-      search,
-    } = this.state;
-
-    if (!mapPreviewedLocationId) return;
+    const { mapLocationData, mapPreviewedLocationId } = this.state;
 
-    if (!mapLocationData[mapPreviewedLocationId]) {
+    if (!mapPreviewedLocationId || !mapLocationData[mapPreviewedLocationId]) {
       // Previewed location has been filtered out, so exit preview mode.
-      this.clearMapPreview();
+      this.mapPreviewSamples = this.samples;
+      return;
+    } else {
+      let conditions = this.getConditions();
+      conditions.filters.locationV2 = [
+        mapLocationData[mapPreviewedLocationId].name,
+      ];
+      this.mapPreviewSamples = this.dataLayer.samples.createView({
+        conditions,
+      });
+    }
+  };
+
+  refreshMapPreviewedProjects = async () => {
+    const { mapLocationData, mapPreviewedLocationId } = this.state;
+
+    if (!mapPreviewedLocationId || !mapLocationData[mapPreviewedLocationId]) {
```

Comment:
When I click the tooltip title to preview a location, I don't see the list actually filter down

![Screen Shot 2019-09-19 at 5 40 46 PM](https://user-images.githubusercontent.com/5652739/65290557-a66ca180-db04-11e9-8de7-31286c46c1ab.png)

Reply:
Fixed.